skcosmo/pcovr/pcovr.py

```diff
@@ -315,4 +480,68 @@ def transform(self, X=None):
 
         """
 
+        check_is_fitted(self, ["pxt_", "mean_"])
+
         return super().transform(X)
+
+    def score(self, X, Y, T=None):
```

Comment:
naming: `score` feels a bit too generic for this function, but I don't know sklearn recommendations here.

Reply:
Again an sklearn naming convention :/